spotinst/bigdata-charts
# BGD-4374 - allow enabling/disabling kill and ghost syncers

## Jira ticket

https://spotinst.atlassian.net/browse/BGD-4374

## Description

 allow enabling/disabling kill and ghost syncers to decrease the load on the service. 

## How to test

pas args:
 - --enable-ghost-syncer=false
 - --enable-kill-syncer=false

the particular syncer should not be started. See logs 

```
{"level":"info","timestamp":"2023-12-05T13:30:45+02:00","message":"Syncer ghost is not enabled"}
```
